actions/host_update_status.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: host_update_status
+pack: zabbix
+runner_type: python-script
+description: Update the status of a Zabbix Host
+enabled: true
+entry_point: host_update_status.py
+parameters:
+    host:
+        type: string
+        description: "Hostname of FQDN of the Zabbix Host"
+        required: True
+    status:
+        type: integer
+        description: "Status to set the Zabbix Host to ex. 0 - monitored host 1 - unmonitored host"
```

Comment:
'ex.' is the wrong terminology here, should be something like "valid values are: "